### PR TITLE
Circle 2 GitHubAction

### PR DESCRIPTION
**Pull request by bam241**
attempt to convert our Circle CI to GithubAction

### CODE REVIEW

**Comment by gonuke** [comment]
This appears to be working, but I'll wait for you to convert it from draft

**Comment by bam241** [comment]
@gonuke I think I am happy with this now.

since your last review, I just removed from circleCI the items I transferred to GitHubAction (build, unit_test, nosetest) and added a news file...

**Review by gonuke — CHANGES_REQUESTED** [review]
This will need a rebase & change to CHANGELOG since I merged the CHANGELOG PR

**Comment by bam241** [comment]
@gonuke I rebased it.

File: .github/workflows/build_test.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        run: |
+          echo "HOME=/github/home/" >> $GITHUB_ENV
+          echo "PATH=$PATH:${HOME}/.local/bin" >> $GITHUB_ENV
+          echo "LD_LIBRARY_PATH=$LD_LIBRARY_PATH:${HOME}/.local/lib" >> $GITHUB_ENV
+      - name: Building Cyclus
+        run: |
+          mkdir -p ${HOME}/.local/lib/python3.7/site-packages/
+          cd ${GITHUB_WORKSPACE}
+          python install.py -j 2 \
+                            --build-type=Release --core-version 999999.999999 \
```
Comment (gonuke) [inline .github/workflows/build_test.yml:40]: Maybe I haven't noticed this before, but what is `core-version` and is this the right one?

**Review by gonuke — APPROVED** [review]
I'm approving because it has apparently been there a while, but with a question about `core-version`

**Comment by gonuke** [comment]
Thanks @bam241

**Pull request merged**